src/sensors/sensor.h

```diff
@@ -96,6 +96,7 @@ class Sensor {
 	const Vector3& getAcceleration() { return acceleration; };
 	const Quat& getFusedRotation() { return fusedRotation; };
 	bool hasNewDataToSend() { return newFusedRotation || newAcceleration; };
+	bool completedRestCalibration();
```

Comment:
If this is only supposed to be called by other subclasses, please move this to the `protected` access level.

Reply:
This is actually a getter, maybe would be better to rename it to "hasCompleted..."?

Reply:
yeah, that'd would make more sense

Reply:
Renamed it and inlined it